bin/devrun.dart

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env dart
+// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// Runs io.js with dev_compiler's generated code.
+library dev_compiler.bin.devrun;
+
+import 'dart:io';
+
+import 'package:dev_compiler/src/options.dart';
+import 'package:dev_compiler/src/runner/file_utils.dart' show turnFilesToUris;
+import 'package:dev_compiler/src/runner/runtime_utils.dart' show
+    listRuntimeFiles, listOutputFiles, getRuntimeFileAlias, getMainModuleName;
+import 'package:dev_compiler/src/runner/iojs_utils.dart' show
+    checkIoJsVersion, startIoJs;
+
+import 'package:path/path.dart';
+
+import 'devc.dart' show validateOptions, compile;
+
+void _showUsageAndExit() {
+  print('usage: dartdevrun [<options>] <file.dart>\n');
+  print('<file.dart> is a single Dart file to run.\n');
+  print('<options> include:\n');
+  print(argParser.usage);
+  exit(1);
+}
+
+main(List<String> args) async {
+  // TODO(ochafik): Should devc accept files anyway?
+  args = turnFilesToUris(args);
```

Comment:
wouldn't we want to do this after we parse options? Otherwise we make system calls to discover "--help" is not really a file. And if it was, then --help would stop working.